app/Domain/Canvas/Repositories/Canvas.php

```diff
@@ -282,7 +282,7 @@ public function getAllCanvas($projectId, $type = null): false|array
          * @param $canvasId
          * @return array|false
          */
-        public function getSingleCanvas($canvasId): false|array
+        public function getSingleCanvas($canvasId)
```

Comment:
Why did you remove that? This is the return value definition and always good to have.